android/app/src/main/kotlin/com/jabook/app/jabook/compose/data/network/DnsPrefetchPolicy.kt

```
@@ -0,0 +1,198 @@
+            val resolved = resolver(host)
+            val elapsed = System.currentTimeMillis() - startedAt
+            synchronized(lock) {
+                hostTimestamps = pruneTimestamps(hostTimestamps, host, currentTimeMs)
+            }
+            PrefetchResult(
+                host = host,
+                success = true,
+                addresses = resolved.map { it.hostAddress.orEmpty() },
+                elapsedMs = elapsed,
+            )
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)